microservices/password-manager/utils/command-utils.ts

```diff
@@ -1,27 +1,32 @@
 import { execSync } from "child_process";
 import path from "path";
 import { existsSync, readFileSync, statSync, writeFileSync } from "fs";
+import { CommandExecutor, CommandOutputType, PipeExecuteStrategy } from "command-executor-lib"
 
 const PIPE_PATH = "/command-runner";
 const PIPE_COMM_DIR = "/pipe-comm";
 const PIPE_OUTPUT_PATH = path.join(PIPE_COMM_DIR, "output.txt");
 const PIPE_OUTPUT_CACHE_MINUTES = 3;
 const PIPE_WAIT_SLEEP_TIME = 100; // milliseconds
 
-const CommandOutputType = {
-  Success: 0,
-  Error: 1,
-};
+// const CommandOutputType = {
+//   Success: 0,
+//   Error: 1,
+// };
 
 class CommandOutput {
-  constructor(type, value) {
+  type: CommandOutputType
+  value: string
+  constructor(type: CommandOutputType, value: string) {
     this.type = type;
     this.value = value;
   }
 }
 
 class PipeCommand {
```

Comment:
The whole point of using command-executor-lib is I moved all of this logic there, this class is no longer required.